class.jetpack-signature.php

```diff
@@ -71,7 +71,17 @@ function sign_current_request( $override = array() ) {
 					$body = $_POST;
 				}
 			}
+		} else if ( 'PUT' == strtoupper( $_SERVER['REQUEST_METHOD'] ) ) {
+			// This is a little strange-looking, but there doesn't seem to be another way to get the PUT body
+			$raw_put_data = file_get_contents( 'php://input' );
```

Comment:
This line gave me pause, because of an earlier conversation here:
https://github.com/Automattic/jetpack/pull/5418#issuecomment-268753286

But I do not think earlier concerns apply as this is for outgoing requests.